rubin_scheduler/scheduler/detailers/dither_detailer.py

```diff
@@ -427,11 +431,15 @@ def __init__(
         dither="night",
         per_night=None,
         seed=42,
-        nnights=7305,
+        nnights=None,
         telescope="rubin",
+        n_in_night=7310,
```

Comment:
This default number continues to not make sense to me. This is the number of dither positions in a single night?